addons/event/static/src/client_action/event_barcode.xml

```diff
@@ -3,35 +3,27 @@
 <templates xml:space="preserve">
 
     <t t-name="event.EventScanView">
-        <div class="o_event_barcode_bg o_home_menu_background">
-            <div class="o_event_barcode_main bg-view">
-                <a href="#" class="o_event_previous_menu float-start"><i class="oi oi-chevron-left" t-on-click.prevent="() => this.onClickBackToEvents()"></i></a>
-                <div class="text-center">
-                    <h1 t-out="data.name"/>
-                    <p>
-                        <t t-if="data.city and data.country">
-                            <t t-out="data.city"/> - <t t-out="data.country"/>
-                        </t>
-                        <t t-if="data.city and !data.country" t-out="data.city"/>
-                        <t t-if="data.country and !data.city" t-out="data.country"/>
-                    </p>
-                    <h2><small>Welcome to</small> <t t-out="data.company_name"/></h2>
-                    <img t-if="data.company_id" t-attf-src="/web/image/res.company/{{data.company_id}}/logo_web" alt="Company Logo" class="o_event_barcode_company_image"/>
-                </div>
-                <div class="row">
-                    <div class="col-sm-5 mt16">
-                        <BarcodeScanner onBarcodeScanned="(ev) => this.onBarcodeScanned(ev)"/>
-                        <h5 class="mt8 mb0 text-muted">Scan a badge</h5>
+        <div class="o_event_barcode_bg o_home_menu_background h-100 w-100 pt-sm-3">
+            <div class="o_event_barcode_main container d-flex flex-column h-100 h-sm-auto bg-view shadow">
+                <header>
+                    <div class="d-flex align-items-center justify-content-between my-3">
+                        <a t-if="!isDisplayStandalone" href="#" class="o_stock_barcode_home_menu oi oi-apps me-3 text-dark" t-on-click="() => this.home.toggle(true)"/>
+                        <span class="fs-2 me-auto ms-2">
+                            <t t-out="data.name"/>
+                        </span>
                     </div>
-                    <div class="col-sm-2 mt32">
-                        <h4 class="mt0 mb8"><i>or</i></h4>
+                </header>
+                <main class="flex-grow-1 d-flex flex-column justify-content-center align-items-center vh-50">
```

Comment:
just a general comment, I see a lot of classes, please double check they are needed (could maybe be simplified)

Reply:
The dom structure should be as close as possible as the one in stable too (if we have something a bit simpler in terms of design it should not be an issue).

Let's try to keep the structure diff as minimal as possible